tests/test_amclient.py

```diff
@@ -863,6 +863,67 @@ def test_extract_individual_file(self):
             assert os.path.getsize(file_) == int(response.get("Content-Length", 0))
             assert filename_to_test in response.get("Content-Disposition", "")
 
+    @vcr.use_cassette("fixtures/vcr_cassettes/microservices.yaml")
+    def test_get_microservices(self):
```

Comment:
new tests! :tada: :tada: :tada:

Reply:
I still find difficult working with pyvcr cassettes. Needing to set record mode first, then adjusting URLs... Probably I'm doing it wrong.

Reply:
I need to look at the API for sure. I do it the same way. I opened an issue in the main repo to say ask around this though, as I also feel it's quite difficult to re-record.